images/build/Dockerfiles/Dockerfile

```diff
@@ -18,9 +18,12 @@ ENV RELEASE_TAG_NAME=${RELEASE_TAG_NAME}
 ENV DEBIAN_FLAVOR="stretch"
 
 RUN ./build.sh dotnetcore /opt/startupcmdgen/dotnet
+RUN ./build.sh golang     /opt/startupcmdgen/golang
+RUN ./build.sh hugo       /opt/startupcmdgen/hugo
 RUN ./build.sh node       /opt/startupcmdgen/nodejs
 RUN ./build.sh php        /opt/startupcmdgen/php
 RUN ./build.sh python     /opt/startupcmdgen/python
+RUN ./build.sh ruby       /opt/startupcmdgen/ruby
```

Comment:
not needed ?

Reply:
Same as https://github.com/microsoft/Oryx/pull/1149/files/ce4f94e5a8fe1c05e3865f38118613f56f67d30d#r746056556

Reply:
same comment

Reply:
@arroyc Ruby currently has startupscriptgenerator code in main: https://github.com/microsoft/Oryx/tree/main/src/startupscriptgenerator/src/ruby
Are we suggesting we remove ruby related startupscriptgenerator code?

Reply:
nah .. i will take back this .. I see ruby startupscript generator is there already .. so you can keep it